src/dispatch/incident/flows.py

```diff
@@ -914,6 +966,20 @@ def incident_assign_role_flow(assigner_email: str, incident_id: int, action: dic
         )
 
     if assignee_role == ParticipantRoleType.incident_commander:
+        # we get the incident document
+        incident_document = get_document(
```

Comment:
@kevgliss if we get the investigation document here, let's not do it again in line 986.